pyteal/ast/abi/__init__.py

```diff
@@ -1,3 +1,5 @@
+from .string import String, StringTypeSpec
```

Comment:
FYI: as of #283 - we're going to be using absolute imports. So this PR may need to follow those conventions, depending on which PR gets merged in first.

Reply:
Additionally, can you please merge in the latest algorand origin master.

Reply:
Please ignore the above suggestions. I see now that this is a _relative_ PR against the ABI branch, and therefore it's up to `feature/abi` to resolve this issue

Reply:
Update: In fact, #284 and #285 do address these issues.